drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/collectioneditor/CollectionViewImpl.java

```diff
@@ -74,22 +80,28 @@
     protected UListElement elementsContainer = Document.get().createULElement();
 
     @DataField("closeCollectionEditorButton")
-    protected ButtonElement closeCollectionEditorButton = Document.get().createButtonElement();
+    protected ButtonElement closeCollectionEditorButton = Document.get().createPushButtonElement();
 
-    @DataField("objectSeparator")
-    protected LIElement objectSeparator = Document.get().createLIElement();
+    @DataField("addItemButtonContainer")
+    protected DivElement addItemButtonContainer = Document.get().createDivElement();
+
+    @DataField("createCollectionRadio")
+    protected InputElement createCollectionRadio = Document.get().createRadioInputElement("collectionRadio");
+
+    @DataField("defineCollectionRadio")
```

Comment:
@yesamer 
What is the difference between those two radio buttons?

Reply:
@gitgabrio The first one is to enables the Create Collection guided editor, which is the "old" way to populate the Collection. The second one enables the Define Collection editor, which consists on a `TextArea` where you can write an expression.